### DISCUSSION TITLE
Error when installing WSA on windows 11

### DISCUSSION THREAD

**Discussion by SohamDoshi** [LSPosed/MagiskOnWSALocal#542] [Q&A] [answered]
Someone tell me what went wrong and what is a solution for this error?
![Screenshot 2023-05-17 215337](https://github.com/LSPosed/MagiskOnWSALocal/assets/106314995/0ad324ee-08e0-4aef-b5e9-e1dd825b9e13)

**Comment by Howard20181** [comment, marked as answer]
Move WSA to a shorter path

**Reply by SohamDoshi** [reply]
Thank you! it worked!!!!
